vertical-pod-autoscaler/pkg/recommender/util/histogram.go

```diff
@@ -277,6 +277,7 @@ func (h *histogram) LoadFromCheckpoint(checkpoint *vpa_types.HistogramCheckpoint
 		h.bucketWeight[bucket] += float64(weight) * ratio
 	}
 	h.totalWeight += checkpoint.TotalWeight
+	h.updateMinAndMaxBucket()
```

Comment:
Can we perhaps add a comment here to explain why we do this? We could reference the issue here for context.

Reply:
Thanks for the suggestion @raywainman. I added some additional comments and also an explanation to the test why those particular values are chosen for the weights. Hopefully, that's helpful.